packages/atomic/cypress/e2e/result-list/result-components/quickview.cypress.ts

```diff
@@ -24,8 +25,7 @@ const addQuickviewInResultList = (props: TagProps = {}) =>
   );
 
 const openModal = () => {
-  cy.wait(5000);
-  QuickviewSelectors.button().click();
+  QuickviewSelectors.button().click({timeout: 8000});
```

Comment:
increase the timout from default value 4000 to 8000 to wait without using cy.wait()